### ISSUE TITLE
Deprecation warnings for encodestring and decodestring

### ISSUE THREAD

**Issue by carlosfunk** [braintree/braintree_python#92]
### General information

* SDK/Library version: 3.37.2
* Language, language version, and OS: Python 3

### Issue description
I've noticed some warnings when running some tests:

```
site-packages/braintree/webhook_notification_gateway.py:26: DeprecationWarning: decodestring() is a deprecated alias since Python 3.1, use decodebytes()
site-packages/braintree/webhook_testing_gateway.py:12: DeprecationWarning: encodestring() is a deprecated alias since 3.1, use encodebytes()
```

**Comment by crookedneighbor** [comment]
Sorry it's taken so long to respond. 

Thanks for the report. We'll get this patched so the deprecation warnings don't appear.

**Comment by jaypatel512** [comment]
The fix is released in [3.39.0](https://github.com/braintree/braintree_python/releases/tag/3.39.0). Please update to latest version. Thank you for pointing out the issue. 🥂
